__tests__/sms.spec.ts

```diff
@@ -1,62 +1,78 @@
+import Sendchamp from "../src";
+import { mobile, publicKey, sender_name } from "../src/config";
+import { SMSRoute, SendchampMode, SenderUseCase } from "../src/constants/types";
 import { SMS } from "../src/services";
 import random from "randomatic";
-import { sendchamp, sender_name } from "../src/test-setup";
```

Comment:
There wasn't any major reason, just that running tests in parallel becomes simpler since each file has its setup also each test file is self-contained and doesn't rely on shared state or setup from other files. This can provide better isolation for tests.

I understand in a case where the test files could increase exponentially, this would be deep-down effect but given it's limited to a few files it should give anyone modifying test ease of update.

Reply:
okay